source/index.js

```diff
@@ -33,7 +33,6 @@ const defaults = {
 		},
 		cache: false,
 		decompress: true,
-		useElectronNet: false,
```

Comment:
Why is this removed?

Reply:
Because it's not needed. See https://github.com/sindresorhus/got/blob/master/source/request-as-event-emitter.js#L43

Reply:
I think it's still useful to declare all options up-front, even if they are `false` by default.

Reply:
Ok, I'll leave that then.

Reply:
Done.